lambdas/user-list-search-corpus-parser-hydration/src/index.ts

```diff
@@ -121,7 +122,11 @@ export function unwrapPayloads(payload: EventPayload): BulkRequestMeta[] {
       messageId: payload.messageId,
     };
     return [parent, ...stories];
-  } else {
+  } else if (
```

Comment:
noting that this is already evaluated before passed to the function; will update in follow-up